modules/fluid_properties/include/userobjects/IdealGasFluidProperties.h

```diff
@@ -20,8 +20,9 @@ class IdealGasFluidProperties : public SinglePhaseFluidProperties
   virtual Real pressure(Real v, Real u) const;
   virtual Real temperature(Real v, Real u) const;
   virtual Real c(Real v, Real u) const;
-  virtual Real cp(Real v, Real u) const;
-  virtual Real cv(Real v, Real u) const;
+  virtual Real cp(Real v=0., Real u=0.) const;
+  virtual Real cv(Real v=0., Real u=0.) const;
```

Comment:
> Did you run into a case where, internal energy and specific volume would not be needed?

Yes, the internal energy and specific volume are never needed to compute cp, cv, and gamma for an ideal gas.  I am not using the Material (as mentioned in another line note) I am using the UserObject directly.  Note: I only changed the default values in the derived class, not the base SinglePhaseFluidProperties case.

So my choices were to either:
* Unnecessarily couple internal energy and specific volume into classes where they are otherwise not needed.
* Call gamma(0., 0.) or with some other "dummy" values every time.
* Add default values to the derived class interface.

I thought the third option was the cleanest one...